src/anemoi/training/losses/weightedloss.py

```diff
@@ -64,30 +65,45 @@ def __init__(
     def add_scalar(self, dimension: int | tuple[int], scalar: torch.Tensor, *, name: str | None = None) -> None:
         self.scalar.add_scalar(dimension=dimension, scalar=scalar, name=name)
 
+    @functools.wraps(ScaleTensor.update_scalar, assigned=("__doc__", "__annotations__"))
+    def update_scalar(self, name: str, scalar: torch.Tensor, *, override: bool = False) -> None:
+        self.scalar.update_scalar(name=name, scalar=scalar, override=override)
+
     def scale(
         self,
         x: torch.Tensor,
         feature_indices: torch.Tensor | None = None,
+        *,
+        without_scalars: list[str] | list[int] | None = None,
     ) -> torch.Tensor:
         """Scale a tensor by the variable_scaling.
 
         Parameters
         ----------
         x : torch.Tensor
             Tensor to be scaled, shape (bs, ensemble, lat*lon, n_outputs)
-        feature_indices:
-            feature indices (relative to full model output) of the features passed in pred and target
+        feature_indices: torch.Tensor, optional
+            feature indices (relative to full model output) of the features/variables passed in pred and target
```

Comment:
Should this be relative to the internal model output? I think we should follow the namespace of the data indices here.

Reply:
I noticed that for the validation metrics, "relative to the model output" is indeed correct. Either way, we should find a way to keep this description general.

Reply:
Agreed, we should keep the names consistent.
This is code transferred from the old losses, so not attached. I'll discuss with @Rilwan-Adewoyin who added this

Reply:
> to keep this description general.

Do you have a suggestion?

Reply:
`feature indices (relative to model output to which this loss is applied) of the features/variables passed in pred and target`

Reply:
I like the new name and docstring. We can update this one as well to `Indices to subset the calculated scalar with, by default None`.